test/controllers/admin/comfy/api_clients_controller_test.rb

```diff
@@ -718,5 +719,124 @@ class Comfy::Admin::ApiClientsControllerTest < ActionDispatch::IntegrationTest
     assert_equal expected_message, flash[:alert]
   end
 
+  # test 'test_rack_timeout_should_throw_timed_out_exception' do
+  #   # Rack::Timeout.stubs(:timeout).returns(1)
+  #   sign_in(@user)
+  #   # get api_namespace_api_clients_url(api_namespace_id: @api_namespace.id)
+  #   get edit_api_namespace_api_client_path(api_namespace_id: @api_client.api_namespace.id, id: @api_client.id)
+  #   sleep(Rack::Timeout.service_timeout + 1)
+  #   assert_response :request_timeout
+
+  #   assert_equal 503, response.status
+
+  #   assert_includes response.body, "Request timeout"
+  # end
+
+  # test 'test_rack_timeout_should_throw_timed_out_exception' do
+  #   assert_raises(Rack::Timeout::RequestTimeoutError) do
+  #     get root_path
+  #     sleep(Rack::Timeout.timeout + 1)
+  #   end
+  # end
+
+
+  # test 'test_rack_timeout_should_throw_timed_out_exception' do
+  #   Rack::Timeout.stubs(:timeout).returns(0.0001)
+
+  #   MyModule::MySinatra.with_fake_route(:get, '/dosomething', ->{ sleep 0.0002 }) do
+  #     get '/dosomething'
+  #   end
```

Comment:
@ajaystha42 any updates on this PR? If its ready for UAT you should remove this commented out code.